### ISSUE TITLE
Does not send 2FA code request

### ISSUE THREAD

**Issue by GoHoos** [davidashman/homebridge-cync-lights#1]
**Describe The Bug:**
While setting up for the first time, entering the email address for a valid account and clicking the “REQUEST 2FA CODE” button does not send the code.

**To Reproduce:**
Attempt to set up the plug in for the first time, fill in an email with a valid account, and click the “REQUEST 2FA CODE” button

**Expected behavior:**
A code should be sent.

**Logs:**

```
There is no information shown in the logs
```

**Plugin Config:**

```json
{
    "platform": "CyncLights",
    "name": "CyncLights"
}
```

**Screenshots:**

**Environment:**

* **Plugin Version**: 1.5.1
* **Homebridge Version**: 1.7.0
* **Node.js Version**: 18.17.0
* **NPM Version**: 
* **Operating System**:  	Raspbian GNU/Linux Bullseye (11)

**Comment by davidashman** [comment]
I'm not able to reproduce this on my real installation or development homebridge.  Can you enable debug logging and try again?  Do you see the blue box in the corner saying that the code was sent?  Can you send me a HAR file from your browser so I can see what communications are happening with the homebridge server?

**Comment by ghost** [comment]
I am experiencing the same issue. 

Plugin version: 1.5.2
Homebridge version: 1.7.0
Node.js version: 20.10.0
Operating system: Windows 10 VM install

Neither the send 2fa nor the submit buttons seem to do anything. The browser network tools show nothing at all when either button is pressed. Referring to your above comment, there is no blue box in the corner. Nothing retains on the logs because it seems the buttons do not actually trigger any sort of event at all.

**Comment by davidashman** [comment]
Found the issue.  Should be fixed in 1.5.3

**Comment by ghost** [comment]
You're the man thank you for your work on this.
